jumpscale/packages/admin/frontend/components/solutions/Info.vue

```diff
@@ -2,10 +2,49 @@
   <div>
     <base-dialog :title="title" v-model="dialog" :loading="loading">
       <template #default>
-        <code-area mode="python" :content="json"></code-area>
+        <v-tabs v-model="tab" background-color="transparent" color="basil" grow>
+          <v-tab :key="title">{{ title }}</v-tab>
+          <v-tab :key="'moredetails'">More details</v-tab>
+
+          <v-tab-item :key="title">
+            <v-simple-table>
+              <template v-slot:default>
+                <tbody>
+                  <tr v-for="(item, key)  in json" :key="key">
+                    <th>{{ key }}</th>
+                    <td
+                      v-if="key === 'Node ids' || key === 'wids' || key === 'Active workload ids'"
```

Comment:
Keys should be defined as an array and check `v-if="array.includes(key)"`, no?
so that we can add other keys later.

Reply:
changed, thanks Ahmed :+1: